### PR TITLE
Fix Bruno Ferraz info and image

### PR DESCRIPTION
**Pull request by bmaferraz**
I updated my info and my image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,21 +152,21 @@ <h5 class="media-heading">André Pinho</h4>
                         <div class="media team-member">
                             <div class="media-left">
                                 <a target="_blank" href="https://github.com/brunomaf">
```
Comment (fntneves) [inline index.html]: This is still using the old username.

File: index.html [hunk 2]
```diff
@@ -152,21 +152,21 @@ <h5 class="media-heading">André Pinho</h4>
...
                             </div>
                             <div class="media-body media-middle">
                                 <h5 class="media-heading">Bruno Ferraz</h4>
                                 <ul class="social-list">
                                     <li class="social-list-item">
-                                        <a target="_blank" href="https://github.com/brunomaf"><i class="fa fa-github"></i></a>
+                                        <a target="_blank" href="https://github.com/bmaferraz"><i class="fa fa-github"></i></a>
                                     </li>
                                     <li class="social-list-item">
                                         <a target="_blank" href="https://twitter.com/bmaf_bm"><i class="fa fa-twitter"></i></a>
```
Comment (fntneves) [inline index.html]: Are you going to change Twitter also?

**Review by fntneves — CHANGES_REQUESTED** [review]
Please change your github link.

**Comment by bmaferraz** [comment]
@fntneves I updated all links. And changed my twitter username according other links.

**Review by fntneves — APPROVED** [review]

**Pull request merged**